Vagrantfile

```diff
@@ -0,0 +1,19 @@
+# Defines our Vagrant environment
+#
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+ # create maven node (change this to new project)
+
+  config.vm.define :maven do |maven_config|
+      maven_config.vm.box = "bento/centos-6.7"
+      maven_config.vm.hostname = "maven"
+      maven_config.vm.network :private_network, ip: "192.168.1.29"
```

Comment:
Private networking is probably overkill here, since there is only a single host.  When I tried to build, it caused a conflict with something, which won't happen without this setting.